Тринадцатая лабораторная работа выполнена

PopovMatvey opened the pull request:
Были выполнены все два задания: 
Реализован калькулятор с использованием контекста и игра "Виселица".
Также, были выполнены все тесты и линтеры.

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Тринадцатая лабораторная работа выполнена

Были выполнены все два задания: 
1. Реализован калькулятор с использованием контекста и игра "Виселица";
2. Были выполнены все тесты и линтеры.

PopovMatvey committed:
исправлен импорт

PopovMatvey committed:
Add files via upload

djilua1 commented:
Ты направил не в ту ветку.

djilua1 commented on src/ex13_js-context/task-02.js:
```diff
@@ -0,0 +1,84 @@
+
+console.log(game.guess('b').guess('a'));
+
+console.log(game.getStatus());
```
Тестовые данные не нужны.

djilua1 commented on src/ex13_js-context/task-02.js:
```diff
@@ -0,0 +1,84 @@
+
+console.log(game.getStatus());
+
+module.exports = Hangman.prototype;
```
Не пишет про поражение, счетчик работает в минус.

djilua1 commented on src/ex13_js-context/task-02.js:
```diff
@@ -0,0 +1,84 @@
+  },
+};
+
+Hangman.prototype = hangman;
```
Давай сразу это засунем в конструктор.

PopovMatvey committed:
Исправления по тринадцатой лабораторной работе

Были исправлены замечания:
1. Удалены тестовые данные;
2. При поражении игра заканчивается
3. Прототип теперь находится в конструкторе;
4. Также. PR я направил не в master, потому что у меня возник конфликт, поэтому направил изменения в резервный master (временно).

PopovMatvey committed:
Delete src/library_task/assets directory

PopovMatvey committed:
Delete index.html

djilua1 commented:
Должны быть только файлы и ченжи, которые относятся к данной работе.
